python/marvin/contrib/vacs/galaxyzoo3d.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+#
+# @Author: José Sánchez-Gallego (gallegoj@uw.edu)
+# @Date: 2018-07-03
+# @Filename: galaxyzoo3d.py
+# @License: BSD 3-clause (http://www.opensource.org/licenses/BSD-3-Clause)
+#
+# @Last modified by: José Sánchez-Gallego
+# @Last modified time: 2018-07-07 14:14:41
+
+
+from __future__ import absolute_import, division, print_function
+
+import astropy
+
+from .base import VACMixIn
+
+
+class GalaxyZoo3DVAC(VACMixIn):
+    """Provides access to the Galaxy Zoo: 3D VAC.
+
+    VAC name: galaxyzoo3d
+
+    URL: https://www.zooniverse.org/projects/klmasters/galaxy-zoo-3d
+
+    Description: Returns people-powered classifications of internal structures
+    of MaNGA galaxies.
+
+    """
+
+    name = 'galaxyzoo3d'
+    version = 'v1_0_0'
+
+    def get_data(self, parent_object):
```

Comment:
Actually I cannot find an entry for galaxyzoo3d in sdss_paths.ini. How is it called?

Reply:
I made a separate branch of tree called `galaxyzoo3d` where we added the name.  Here is the entry we added in that branch, `mangagalaxyzoo3d = $MANGA_SANDBOX/galaxyzoo3d/{gz3dver}/{mangaid}_{ifusize}_{zooid}.fits.gz`

Reply:
Is there a definition of how `zooid` is calculated in `sdss_access`? Otherwise this definition becomes harder to use than a self-generated path.

Reply:
`zooid` isn't calculated.  If I recall, that is a random number Coleman made up.  Or it may have some internal significance but that's it.  In his code, it's an input.  Users would have to look this number up whether they generated the path themselves or used sdss_access.  We might not need the `zooid`.  The `mangaid` and `ifusize` is enough to pick out a unique file.  Setting `zooid='*'` would probably work.  We should double-check with Coleman though.

Reply:
Great. Random numbers in paths. Something else? A sphinx asking you three questions before you can download. Hurdles?